app/services/langfuse/tasks/score_monitor/score.py

```
@@ -291,14 +291,27 @@ async def send_score_alert(
             fallback_text=f"LLM Judge Failure: {evaluator_name} - Score 0.0",
         )
 
+        # Track alert for daily summary if sent successfully
+        if alert_sent:
+            try:
+                from app.services.langfuse.tasks.alert_summary.summary import (
+                    daily_alert_summary,
+                )
+
+                await daily_alert_summary.track_alert_sent(evaluator_name)
+            except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)